### DISCUSSION TITLE
stm32f091cc supported?

### DISCUSSION THREAD

**Discussion by gigawatts** [stm32duino/Arduino_Core_STM32#1705] [General]
Is the stm32f091cct6 variant supported? I see the stm32f091RC and stm32f091RB are in the list.

**Comment by fpistm** [comment]
Hi,
STM32F091CC has been not declaredin the boards.txt but can be added easily as all generated file are available
https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32F0xx/F091C(B-C)(T-U)

You can follow the wiki to declare it:
https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-%28board%29
